src-java/stats-topology/stats-storm-topology/src/main/java/org/openkilda/wfm/topology/stats/bolts/metrics/FlowMetricGenBolt.java

```diff
@@ -51,19 +56,32 @@ protected void handleInput(Tuple input) throws Exception {
         long timestamp = pullContext(input).getCreateTime();
         SwitchId switchId = stats.getSwitchId();
 
-        for (FlowStatsAndDescriptor entry : stats.getStatsEntries()) {
-            handleStatsEntry(entry.getData(), timestamp, switchId, entry.getDescriptor());
+        List<FlowStatsAndDescriptor> statsEntries = stats.getStatsEntries();
+        Map<Long, Boolean> isAlreadySentCookieMap = statsEntries.stream()
+                .collect(Collectors.toMap(key ->
+                        key.getData().getCookie(), value -> false, (v1, v2) -> v1, HashMap::new));
+
+        for (FlowStatsAndDescriptor statsEntry : statsEntries) {
+            handleStatsEntry(statsEntry.getData(), timestamp, switchId, statsEntry.getDescriptor(),
+                    useAnyFlowHandlerForCookie(isAlreadySentCookieMap, statsEntry));
+            isAlreadySentCookieMap.put(statsEntry.getData().getCookie(), true);
         }
     }
 
+    private static boolean useAnyFlowHandlerForCookie(Map<Long, Boolean> isAlreadySentCookieMap,
+                                                      FlowStatsAndDescriptor statsEntry) {
+        return !isAlreadySentCookieMap.get(statsEntry.getData().getCookie());
+    }
+
```

Comment:
Here is a dirty fix, please, suggest an alternatives solution if any.
Here we have isAlreadySentCookieMap, that contains the mapping between all the cookies from the stats.getStatsEntries and flags, that represents wether this cookie already been sent to the AnyFlowHandler(to generate raw entries).
And the method useAnyFlowHandlerForCookie() that check wether this cookie been sent. 
This fix was done to avoid sending duplicated RAW metrics for the same cookie, same switchId, but for the different measure points.